src/main/java/ru/training/at/hw8/constants/SingleValueParameter.java

```diff
@@ -0,0 +1,19 @@
+package ru.training.at.hw8.constants;
+
+import ru.training.at.hw8.propertyloader.PropertyLoader;
+
+import java.util.Properties;
+
+import static ru.training.at.hw8.propertyloader.PropertyLoader.*;
+
+public class SingleValueParameter {
+    private static final Properties envProperties = getProperties();
+    public static final String KEY;
+    public static final String TOKEN;
+
+    static {
```

Comment:
Why static?

Reply:
I made all parameters static final, to use static imports and because it does not change during execution. Is it incorrect?

Reply:
Why should not I use static?

Reply:
And static, because I do not need class instances

Reply:
I do not need different values of variables for different class instances